docs/docusaurus/docs/core/define_expectations/test_an_expectation.md

```diff
@@ -0,0 +1,189 @@
+---
```

Comment:
I think this file reads pretty well. I might consider shortening the first snippet and just assume the user has a batch definition, but potato patahto